lib/pacio_test_kit/common_tests/create_test.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../interactions_test'
+
+module PacioTestKit
+  class CreateTest < Inferno::Test
+    include PacioTestKit::InteractionsTest
+
+    title 'Server makes correct resource from create interaction'
+    id :pacio_resource_create
+    description 'A server SHALL support the create interaction.'
+
+    input :resource_list,
```

Comment:
We do not need this input. we should update the description of this test to let the user know how this test work. e.g. Inferno will performed a create interaction by `POST`ing a valid resource for the given group. something like that. The resource to use for create can be passed as `config.options`